app/lib/efile/benefits_eligibility.rb

```diff
@@ -53,5 +66,35 @@ def odc_amount
 
       dependents.count { |d| !d.qualifying_ctc? && (d.qualifying_child? || d.qualifying_relative?) } * 500
     end
+
+    def outstanding_recovery_rebate_credit
+      if year == 2020
+        return nil unless intake.eip1_amount_received && intake.eip2_amount_received
+
+        outstanding_eip1 + outstanding_eip2
+      elsif year == 2021
+        return nil unless intake.eip3_amount_received
+
+        outstanding_eip3
+      end
+    end
+
+    def claimed_recovery_rebate_credit
+      return 0 if intake.claim_owed_stimulus_money_no?
+
+      outstanding_recovery_rebate_credit
+    end
+
+    private
+
+    def rrc_eligible_filer_count
+      return intake.primary_tin_type == "ssn" ? 1 : 0 if tax_return.filing_status_single?
```

Comment:
Should we also check for the presence of an ssn in addition to tin_type; Idk are there scenarios where someone has a tin_type but no ssn?